controllers/administration.js

```diff
@@ -1020,42 +970,54 @@ router.get(
 			$populate: ['year'],
 			$sort: 'displayName',
 		});
-		const accountPromise = api(req).get('/accounts/', {
-			qs: { userId: req.params.id },
-		});
+		const accountPromise = api(req, { json: true, version: 'v3' })
+			.get('/account/', {
+				qs: {
+					type: 'userId',
+					value: req.params.id,
+				},
+			});
 
 		Promise.all([
 			userPromise,
 			classesPromise,
 			accountPromise,
-		]).then(([user, _classes, _account]) => {
-			const account = _account[0];
-			const hidePwChangeButton = !account;
+		])
+			.then(([user, _classes, _account]) => {
+				let account;
+				if (_account && _account.total > 0) {
+					account = _account.data[0];
+				}
 
-			const classes = _classes.map((c) => {
-				c.selected = c.teacherIds.includes(user._id);
-				return c;
-			});
-			const canDeleteUser = res.locals.currentUser.permissions.includes('TEACHER_DELETE');
-			res.render('administration/users_edit', {
-				title: res.$t('administration.controller.link.editTeacher'),
-				action: `/administration/teachers/${user._id}`,
-				submitLabel: res.$t('global.button.save'),
-				closeLabel: res.$t('global.button.cancel'),
-				user,
-				canDeleteUser,
-				consentStatusIcon: getConsentStatusIcon(user.consentStatus, true),
-				consent: user.consent,
-				classes,
-				editTeacher: true,
-				hidePwChangeButton,
-				isAdmin: res.locals.currentUser.permissions.includes('ADMIN_VIEW'),
-				schoolUsesLdap: res.locals.currentSchoolData.ldapSchoolIdentifier,
-				referrer: req.header('Referer'),
-				hasAccount: !!account,
-				accountId: account ? account._id : null,
+				const hidePwChangeButton = !account;
+
+				const classes = _classes.map((c) => {
+					c.selected = c.teacherIds.includes(user._id);
+					return c;
+				});
+				const canDeleteUser = res.locals.currentUser.permissions.includes('TEACHER_DELETE');
+				res.render('administration/users_edit', {
+					title: res.$t('administration.controller.link.editTeacher'),
+					action: `/administration/teachers/${user._id}`,
+					submitLabel: res.$t('global.button.save'),
+					closeLabel: res.$t('global.button.cancel'),
+					user,
+					canDeleteUser,
+					consentStatusIcon: getConsentStatusIcon(user.consentStatus, true),
+					consent: user.consent,
+					classes,
+					editTeacher: true,
+					hidePwChangeButton,
+					isAdmin: res.locals.currentUser.permissions.includes('ADMIN_VIEW'),
+					schoolUsesLdap: res.locals.currentSchoolData.ldapSchoolIdentifier,
+					referrer: req.header('Referer'),
+					hasAccount: !!account,
+					accountId: account ? account._id : null,
```

Comment:
This will not work, because 'account._id' was renamed with api v3 to 'account.id'.

Reply:
Done